src/components/Header/Header.css

```diff
@@ -0,0 +1,13 @@
+.Header{
+	overflow: auto;
+}
+.Header.dark{
+	background: #333;
+	color: #fff;
+}
```

Comment:
Is ``dark`` a class we're going to be using across the whole site? Maybe we should do .Header--dark instead? Although this would mean having to do <div className="Header Header--dark"> but such is the bane of BEM